### PR TITLE
NC | NSFS | CLI | Improve Performance of List with Name Filter

### PR DESCRIPTION
**Pull request by shirady**
### Explain the changes
1. In `list_config_files` "mock" the entries array to have one entry and it is the name by the filter (for performance).
2. Exclude the action list from `fetch_bucket_data` and `fetch_account_data`.
3. Edit the account list tests to validate that we receive an array in the response.

### Issues: 
1. Partial fix for #8264 . 

#### Unit Tests:
Please run: `sudo npx jest test_nc_nsfs_account_cli.test.js`.

#### Manual Tests:
1. Create an account with the CLI: `sudo node src/cmd/manage_nsfs account add --name shira-1001 --new_buckets_path /tmp/nsfs_root1 --access_key <access-key> --secret_key <secret-key> --uid <uid> --gid <gid>`
Note: before creating the account need to give permission to the `new_buckets_path`: `chmod 777 /tmp/nsfs_root1`.
2. Account list with decryption and filter name: `sudo node src/cmd/manage_nsfs account list --wide --show_secrets --name <account-name>` (should still work).

- [ ] Doc added/updated
- [X] Tests added

### CODE REVIEW

File: src/cmd/manage_nsfs.js [hunk 1]
```diff
@@ -83,7 +83,10 @@ async function main(argv = minimist(process.argv.slice(2))) {
 }
 
 async function bucket_management(action, user_input) {
-    const data = await fetch_bucket_data(action, user_input);
+    let data;
+    if (action !== ACTIONS.LIST && action !== ACTIONS.STATUS) {
```
Comment (romayalon) [inline src/cmd/manage_nsfs.js:87]: for list it's great, for status I would go with the general flow

File: src/cmd/manage_nsfs.js [hunk 2]
```diff
@@ -567,12 +570,19 @@ function filter_bucket(bucket, filters) {
...
+async function list_config_files(type, wide, show_secrets, filters) {
+    if (wide && Object.keys(filters).length === 1 && filters.name !== undefined) {
+        if (type === TYPES.ACCOUNT) {
+            return get_account_status(filters, show_secrets);
+        } else { // type === TYPES.BUCKET
+            return get_bucket_status(filters);
+        }
+    }
+
     const entries = type === TYPES.ACCOUNT ?
```
Comment (romayalon) [inline src/cmd/manage_nsfs.js:586]: if (filters?.name !== undefined) {
    entries = [{'name': filters.name}]
}

File: src/cmd/manage_nsfs.js (hunk 1 above)
Comment (shirady) [reply src/cmd/manage_nsfs.js]: Changed, thanks.

**Review by romayalon — APPROVED** [review]
LGTM, approving
minor thing - there is an indentation issue on line 582

**Pull request merged**